components/hab/src/command/studio/docker.rs

```diff
@@ -339,4 +364,30 @@ mod tests {
                    format!("{}-{}:{}-{}",
                            DOCKER_WINDOWS_IMAGE, "x86_64-windows", "ltsc2016", VERSION));
     }
+
+    #[test]
+    fn update_ssl_cert_file_envvar_not_set() {
+        let mnt_prefix = "";
+        let env_var = lock_ssl_cert_file_env_var();
+        env_var.unset();
+
+        update_ssl_cert_file_envvar(mnt_prefix);
+        assert!(std::env::var(SSL_CERT_FILE_ENVVAR).is_err());
+    }
+
+    #[test]
+    fn update_ssl_cert_file_envvar_has_value() {
+        let mnt_prefix = "";
+        let key_name = "ssl-test-cert.pem";
+
+        let env_var = lock_ssl_cert_file_env_var();
+        env_var.set(format!("/path/to/{}", key_name));
+
+        update_ssl_cert_file_envvar(mnt_prefix);
+        let internal_cert_path = Path::new(CACHE_SSL_PATH).join(key_name)
+                                                          .to_str()
+                                                          .unwrap()
+                                                          .to_string();
+        assert_eq!(std::env::var(SSL_CERT_FILE_ENVVAR), Ok(internal_cert_path));
```

Comment:
This test would be more valuable if `mnt_prefix` weren't an empty string. As is it is now, we can break `update_ssl_cert_file_envvar` by changing
```rust
env::set_var(SSL_CERT_FILE_ENVVAR,
             Path::new(mnt_prefix).join(CACHE_SSL_PATH)
                                  .join(cert_file_name));
```
to
```rust
env::set_var(SSL_CERT_FILE_ENVVAR,
             Path::new(CACHE_SSL_PATH).join(cert_file_name));
```
and the test still passes